### ISSUE TITLE
QA Finding: role="alert" on Confirmation page should not encompass "Go back to submissions" link

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#129841]
## Expected vs. Actual
**Expected**:  

`role="alert"` only encompasses the `va-alert` component.

**Actual**: 

`role="alert"` is placed on the div encompassing both the alert and the "Go back to submissions page" link, making the alert announcement more verbose than it needs to be.

## Steps to Reproduce

1. Go through the ITF flow, but trigger the failure to submit state
2. Observe the `role="alert"` implementation

## Examples

<img width="834" height="847" alt="Image" src="https://github.com/user-attachments/assets/83447468-4cf4-4132-a731-1031a3a27f10" />
